app/api-v1/services/attachmentService.js

```diff
@@ -5,7 +5,7 @@ const { HttpResponseError } = require('../../utils/errors')
 const createAttachmentFromFile = async (file) => {
   return new Promise((resolve) => {
     fs.readFile(file.path, async (err, data) => {
-      if (err) throw new HttpResponseError({ code: 500, message: err.message, service: 'attachment' })
+      if (err) throw new HttpResponseError({ code: 500, message: err.message, file })
```

Comment:
`file` doesn't get handled in `HttpResponseError`

Reply:
Yea, looks like the assumption was made that this is a second argument rather than an object. Thanks